docs/alert-processing.md

```diff
@@ -0,0 +1,72 @@
+# Alert processing
+
+BOOM consumes Kafka streams of alerts from astronomical surveys
+and outputs Kafka streams for consumers like SkyPortal.
+
+Each alert is processed with the following pipeline:
+
+1. Cross-matches with object IDs from other data catalogs are added.
```

Comment:
While it definitely is one of the important steps carried by the alert worker, it might seem like an oversimplification to only talk about crossmatching. If you look at the alert workers a lot of the work we do is standardize the data we receive from each survey to unify the schemas where applicable, format it, split it for each table, ... and crossmatching with archival catalogs (which is omitted here btw) and other alert-based surveys. This should definitely be mentioned in here! Maybe https://docs.google.com/presentation/d/1PGpJQn2qcohc8fVJQqpfcVRy8ZKZmqF0J7xYygQOmWk/edit?usp=sharing can help with figuring out the content of this markdown document.

Reply:
Gotcha. I will definitely add the normalization stuff.